### PR TITLE
feat: cart-management

### PR DESCRIPTION
**Pull request by KaustubhM1997**

### CODE REVIEW

File: pages/cart-management/cart-management.css [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  font-size: 1.2rem;
+}
+
+.product-card {
+  display: flex;
+  margin-top: 2rem;
+  border: 1px solid;
+  gap: 6rem;
+  height: 13rem;
+  box-shadow: 0px 0px 5px 5px var(--light-grey);
```
Comment (Chakravarthi0) [inline pages/cart-management/cart-management.css]: You are using the same value in price-details class. You can create a variable if you are using this value multiple places.

**Review by Chakravarthi0 — APPROVED** [review]
looks good to me

File: pages/cart-management/cart-management.css [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+
+:root {
+  --light-grey: #9ca3af;
+  --button-hover: #be185d;
+  --text-green: #16a34a;
+  --box-shadow: 0px 0px 5px 5px var(--light-grey);
+  --border-default: 1px solid;
+}
+
```
Comment (adidoshi) [inline pages/cart-management/cart-management.css:11]: Just a suggestion, if we can shift `:root` to main.css then you can use it everywhere.

**Review by adidoshi — APPROVED** [review]
Rest looks good!!

**Pull request merged**